packages/server/tests/localAcceptance.env

```diff
@@ -27,6 +27,6 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
-HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_RATE_LIMIT_TINYBAR=12000000000 # 100 HBARs
```

Comment:
Why is there a need to increase these variables? This could potentially cause the test `should eventually exhaust the HBAR limit for a BASIC user after multiple deployments of large contracts` to fail. The total budget might be too high, and the duration too short, which could result in the budget resetting during the deployment loop. Additionally, with a higher `HBAR_RATE_LIMIT_TINYBAR`, the exhaustion test could take much longer than necessary.

I’d suggest reverting it back. If you believe the change is necessary, then the duration should be updated accordingly to prevent the limiter from resetting during the deployment loop. Another option is to override these variables specifically for the `acceptancetest:hbarlimiter` npm command. Something like this:

```bash
"acceptancetest:hbarlimiter": "HBAR_RATE_LIMIT_TINYBAR=5000000000 nyc ts-mocha packages/server/tests/acceptance/index.spec.ts -g '@hbarlimiter' --exit",
```

Reply:
Leftovers from testing, reverted to the values defined in the `main` branch.